### ISSUE TITLE
PSReadline on Mac does not allow to open pwsh-preview

### ISSUE THREAD

**Issue by nippyin** [PowerShell/PSReadLine#2633]
## Environment

```none
[run the script below and paste the output here]
```

## Exception report

Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta2PowerShell: 7.2.0-preview.7
OS: Darwin 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:33 PDT 2021; root:xnu-7195.121.3~9/RELEASE_X86_64
BufferWidth: 92
BufferHeight: 56

Last 0 Keys
```

```

### Exception```System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004ca+0x34
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000101+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000fa+0x371
```

## Steps to reproduce

push-preview on terminal in macOS

## Expected behavior

![image](https://user-images.githubusercontent.com/41337248/124118949-e84bab00-da3f-11eb-859a-419e38d19968.jpeg)

## Actual behavior

PowerShell does not load.

**Comment by SteveL-MSFT** [comment]
This is fixed in 2.2-beta3

**Comment by nippyin** [comment]
Thanks Much, it works.



________________________________
